app/views/games/index.html.erb

```diff
@@ -26,8 +26,22 @@
       <div class="small-4 large-centered columns">
         <h3>Join one of these games</h3>
       </div>
-        <p> (list of all games. Have to add button to take user to that game still)
-        <h1><%= @games.inspect %></h1>
+
+
+        <div class="row">
+          <div class="small-4 small-centered columns">
+            <ul>
+              <% @games.each do |game| %>
+                <li>
+                  <%= game.name %>
+                  <%= link_to 'Join Game', game_path(game), :class => 'custom-button-class' %>
```

Comment:
Hey Aaron, I'm stuck. I can't figure out the code to get the user to join the game. 

I think it's <%= link_to 'Join Game', game_path(game),  :method => 'PATCH' %>

But when I do this, I get "Template is missing" Error. I've been looking up a way to fix this, and need a little guidance.